src/Components/App/App.js

```diff
@@ -1,13 +1,15 @@
 
 import './App.css';
 import {Route, Routes} from 'react-router-dom'
+import Category from '../Category/Category'
+import Difficulty from '../Difficulty/Difficulty'
 
 function App() {
   return (
     <div className="App">
       <Routes>
-        <Route path="/" element={<h1>category component</h1>}/>
-        <Route path=':category' element={<h1>difficulty component</h1>} />
+        <Route path="/" element={<Category />}/>
+        <Route path=':category' element={<Difficulty />} />
         <Route path=':category/:difficulty' element={<h1>option component</h1>} />
```

Comment:
I'm not certain if it will break anything down the road by not including it, but I think lines 12 & 13 should have a `/` before `:category`